website/docs/Software/OpenBCISoftware/NeuroFly_Toolkit.md

```diff
@@ -12,6 +12,9 @@ This tutorial will show you how to control a joystick using EMG data with the Op
 3.  [EMG/ECG Snap Electrode Cables](https://shop.openbci.com/collections/frontpage/products/emg-ecg-snap-electrode-cables?variant=32372786958)
 4.  Computer with downloaded [OpenBCI GUI](Software/OpenBCISoftware/01-OpenBCI_GUI.md). **Be sure to use [OpenBCI GUI v5.2.0](https://github.com/OpenBCI/OpenBCI_GUI/releases/latest) or later**.
 
+## Video Explainer
```

Comment:
Can we rename this heading?

Reply:
Yes. I changed it in a new commit, but feel free to make it something else